app/src/main/java/org/thoughtcrime/securesms/onboarding/messagenotifications/MessageNotifications.kt

```diff
@@ -90,8 +100,58 @@ internal fun MessageNotificationsScreen(
 
 @Composable
 private fun NotificationRadioButton(
```

Comment:
With this method you now have two components defined separately that should be doing the same thing, which isn't very scalable. You'd have to modify both every time something changes.
You should instead:
- Either keep both definition, but the stringRes one should then call the second implementation directly, that way it isn't redefined (it would call the other implementation and call stringResource in there).
- Or remove the stringRes implementation and only keep the one with Strings.

Reply:
Fair. Have adjusted the resource ID version to pass through to the strings version.